firebase-app-distribution/src/main/java/com/google/firebase/app/distribution/InstallActivity.java

```diff
@@ -25,13 +25,16 @@
 import java.io.File;
 
 /**
- * Activity opened during installation in {@link UpdateApkClient} after APK download is finished.
+ * Activity opened during installation in {@link FirebaseAppDistribution} after APK download is
+ * finished.
  */
-public class InstallActivity extends AppCompatActivity {
+class InstallActivity extends AppCompatActivity {
   private static final String TAG = "InstallActivity: ";
   private boolean unknownSourceEnablementInProgress = false;
   private boolean installInProgress = false;
 
+  public InstallActivity() {}
```

Comment:
Why do we need a public default constructor here?

Reply:
This can be ignored. This must also be public mistake on my part.